homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -194,38 +202,21 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 
     async_add_entities([mirobo], update_before_add=True)
 
-    async def async_service_handler(service):
+    async def async_service_handler(entity, service):
```

Comment:
We don't need this service handler since we just want to pass the params that are not entity id to the entity method when calling that.

We can just register the services like this as in the example in the docs:
```py
# This will call Entity.set_sleep_timer(sleep_time=VALUE)
platform.async_register_entity_service(
    SERVICE_SET_TIMER,
    {
        vol.Required('sleep_time'): cv.time_period,
    },
    "set_sleep_timer",
)
```

Reply:
Cool, I changed it.